package-lock.json

```diff
@@ -9790,7 +9790,7 @@
       "dependencies": {
         "async": {
           "version": "1.5.2",
-          "resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
+          "resolved": "http://registry.npmjs.org/async/-/async-1.5.2.tgz",
```

Comment:
What's wrong with https?

Reply:
I don't know why it keeps changing whenever I do `npm install`.

Reply:
Yeah, we've encountered this before. Until we figure it out, you should just change this file back.